src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -462,6 +462,16 @@ impl TargetInfo {
                     // preserved.
                     should_replace_hyphens: true,
                 })
+            } else {
+                ret.push(FileType {
+                    suffix: ".dwp".to_string(),
+                    prefix: prefix.clone(),
+                    flavor: FileFlavor::DebugInfo,
+                    crate_type: Some(crate_type),
+                    // Currently no known reason to prefer one naming scheme
```

Comment:
Maybe we could have a comment with a link to <https://dwarfstd.org/doc/DWARF5.pdf> as a reference, like what `.pdb` and `.dSYM` do.